HighGear Workflow Connector

dhands5312 opened the pull request:
---
### HighGear Workflow Connector
This is the initial submission of the HighGear Workflow connector for certification.

The following files are being added:
- apiDefinition.swagger.json
- apiProperties.json
- readme.md

The following items have been verified:
- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.
  - HighGear brand color is `#404545`

dhands5312 committed:
Adding the HighGear Workflow connector files for certification

dhands5312 commented:
@microsoft-github-policy-service agree company="HighGear Software, Inc."

Amjed-Ayoub commented on certified-connectors/HighGear Workflow/apiDefinition.swagger.json:
```diff
@@ -0,0 +1,2050 @@
+          "Task"
+        ],
+        "summary": "Get task field schema",
+        "operationId": "Task_GetTaskSchema",
```
@dhands5312 operationIds should be in Camel Case. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please change to "TaskGetTaskSchema". Please fix all operationIds in the swagger file.

dhands5312 replied:
@Amjed-Ayoub I have corrected all the operationIds to use Camel Case.

dhands5312 committed:
Changed operationIds to use Camel Case
- Removed the '_' between controller name and action
- Fixed Titles for Contact properties
- Added descriptions for Contact properties
- Corrected typo

Amjed-Ayoub reviewed (approved):
Hello @dhands5312 ,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.
